Classes/Private/Manager/Database/OCTRealmManager.h

```diff
@@ -51,6 +53,7 @@
 
 - (OCTFriend *)friendWithFriendNumber:(OCTToxFriendNumber)friendNumber;
 - (OCTChat *)getOrCreateChatWithFriend:(OCTFriend *)friend;
+- (OCTCall *)getOrCreateCallWithFriendNumber:(OCTToxFriendNumber)friendNumber;
```

Comment:
I think it would be better to pass `friend` as a parameter instead of `friendNumber`, to be consistent with chat method. The reason behind this is that quite often the caller of the method already have a friend object. Also in that case there is no need in getting friend in implementation.